graph/src/components/store.rs

```diff
@@ -41,6 +43,26 @@ pub enum EntityFilter {
     NotEndsWith(Attribute, Value),
 }
 
+// Define some convenience methods
+impl EntityFilter {
+    pub fn new_equal(
```

Comment:
I think these could both be just `in` and `equal`. What do you think?

Reply:
Unfortunately `in` is a reserved keyword :(

Reply:
Ok, nevermind!